test/unit/construction/ConfigConstructorPool-test.ts

```diff
@@ -285,6 +286,10 @@ describe('ConfigConstructorPool', () => {
           types: 'ex:Component',
         }), creationSettings);
         expect(instance1).not.toBe(instance2);
+        await expect(pool.getInstanceRegistry()['ex:myComponentInstance1']).resolves.toBe('INSTANCE0');
+        await expect(pool.getInstanceRegistry()['ex:myComponentInstance2']).resolves.toBe('INSTANCE1');
+
+        // Expect('ex:myComponentInstance2' in pool.getInstanceRegistry()).toBeTruthy();
```

Comment:
I suspect this line can go?

Reply:
Indeed, that should not have been there anymore. Force pushing change